src/app/generic-listing-page/generic-listing-page.component.ts

```diff
@@ -66,11 +68,16 @@ export class GenericListingPageComponent  implements OnInit {
     this.noData=true;
     await this.loaderService.showLoading("LOADER_MSG")
     let url = `${this.pageConfig.apiUrl}&page=${this.page}&limit=${this.limit}&searchText=${this.searchTerm}${this.filterQuery}`
-    this.projectsApiService.get(url).subscribe({
+    let serviceToTrigger = this.projectsApiService
+    if(this.pageConfig?.type == "recommendation"){
+      url = `${this.pageConfig.apiUrl}?page=${this.page}&limit=${this.limit}`
+      serviceToTrigger = this.gwApiService 
+    }
+    serviceToTrigger.get(url).subscribe({
       next: async(response: any)=>{
       await this.loaderService.dismissLoading()
       this.noData=false;
-      if(response.status == 200){
+      if(response){
```

Comment:
Hope this doesn't cause issues when the response is other than 200

Reply:
This works, but inside if block I will handle it